### PR TITLE
Fixing eslint issues in widgets source files

### PR DESCRIPTION
**Pull request by Ambujaa10**

### CODE REVIEW

File: components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/package.json [hunk 1]
```diff
@@ -22,7 +22,8 @@
     "test": "react-scripts test --env=jsdom",
     "eject": "react-scripts eject",
     "clean": "rimraf dist",
-    "dev": "NODE_ENV=development webpack -d --config webpack.config.js --watch"
+    "dev": "NODE_ENV=development webpack -d --config webpack.config.js --watch",
+    "build-dev": "webpack -d --config webpack.config.js --watch --progress"
```
Comment (this) [inline components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/package.json]: Why have both `dev` and `build-dev`?

Comment (this) [reply components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/package.json]: Are we using webpack dev server here? If not, then no point of having `--watch` flag.

Comment (this) [reply components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/package.json]: Better to use this `dev` script https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards-web-component/package.json#L9

File: components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/.eslintrc [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+      ],
+      "jsx-quotes": [
+        "off"
+      ],
+      "no-else-return": "off"
+    },
+    "plugins": [
+      "react"
+    ]
+  }
```
Comment (this) [inline components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/.eslintrc]: Hope this was taken from here https://github.com/wso2/carbon-dashboards/blob/master/components/dashboards-web-component/.eslintrc

Please add file-ending new line.

File: components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsSessionCount/,eslintrc [hunk 3]
```diff
@@ -0,0 +1,113 @@
+{
```
Comment (this) [inline components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsSessionCount/,eslintrc]: File name of this fie is `,eslintrc` (note comma), not `.eslintrc`

File: components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsSessionCount/src/IsAnalyticsSessionCount.jsx [hunk 4]
```diff
@@ -61,50 +62,49 @@ class IsAnalyticsSessionCount extends Widget {
...
-        let dataProviderConfigs = _.cloneDeep(this.state.providerConfig);
-        let query = dataProviderConfigs.configs.config.queryData.query;
-        for(let i=0; i<5; i++) {
-        query = query
-            .replace("{{from}}", this.state.fromDate)
-            .replace("{{to}}", this.state.toDate)
-            .replace("{{now}}", new Date().getTime());
+        const dataProviderConfigs = _.cloneDeep(this.state.providerConfig);
+        let { query } = dataProviderConfigs.configs.config.queryData;
+        for (let i = 0; i < 5; i++) {
```
Comment (this) [inline components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsSessionCount/src/IsAnalyticsSessionCount.jsx]: Why `5`?

DOn't use magic numbers: https://stackoverflow.com/questions/47882/what-is-a-magic-number-and-why-is-it-bad

File: components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/src/IsAnalyticsAverageSessionDuration.jsx [hunk 5]
```diff
@@ -59,7 +60,7 @@ class IsAnalyticsAverageSessionDuration extends Widget {
             width: this.props.glContainer.width,
```
Comment (this) [inline components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/src/IsAnalyticsAverageSessionDuration.jsx]: Please rename `ChartConfig` property to `chartConfig` in above line (58)

File: components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/src/IsAnalyticsAverageSessionDuration.jsx [hunk 6]
```diff
@@ -123,12 +124,12 @@ class IsAnalyticsAverageSessionDuration extends Widget {
 
     assembleQuery() {
         super.getWidgetChannelManager().unsubscribeWidget(this.props.id);
-        let dataProviderConfigs = _.cloneDeep(this.state.providerConfig);
-        let query = dataProviderConfigs.configs.config.queryData.query;
+        const dataProviderConfigs = _.cloneDeep(this.state.providerConfig);
```
Comment (this) [inline components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/src/IsAnalyticsAverageSessionDuration.jsx]: I don't see `providerConfig` is being set to the `state` in the constructor. If you are putting something to the `this.state`, then it should be set in the constructor. (if you don't have a value then set a default value or `null`)

File: components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/.eslintrc (hunk 2 above)
Comment (this) [reply components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/.eslintrc]: Applies for other places as well.

File: components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/src/IsAnalyticsAverageSessionDuration.jsx (hunk 6 above)
Comment (this) [reply components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/src/IsAnalyticsAverageSessionDuration.jsx]: @Ambujaa10 Is this been addressed?

Comment (Ambujaa10) [reply components/is-analytics/org.wso2.analytics.solutions.is.analytics/widgets/IsAnalyticsAverageSessionDuration/src/IsAnalyticsAverageSessionDuration.jsx]: yes. I assigned providerConfig to null in this.state

**Pull request merged**